app/assets/stylesheets/admin/helpers/_broken_links_report.scss

```diff
@@ -5,3 +5,36 @@
     }
   }
 }
+
+.broken-links-report {
+  .issue-fix {
+    font-weight: normal;
+  }
+
+  .issue-list {
+    list-style: none;
+    padding-left: 0;
+    li {
+      margin-top: 1rem;
+    }
+  }
+
+  .issue-summary {
+    display: block;
+    font-weight: bold;
+    padding-top: 1rem;
+  }
+
+  .issue-summary::-webkit-details-marker {
```

Comment:
Can you explain what this is doing? `::-webkit-details-marker` looks to be removing the triangle that indicates if the detail element is open or close, but why do we need to remove and re add it?

Reply:
I think webkit is the only browser to implement the drop down arrow by default, so by hiding it we are able to give a more consistent cross browser look and feel